fr-French/Localizable.strings

```diff
@@ -58,241 +58,241 @@
+"reset_password" = "Réinitialisation du mot de passe";
+"reset_password_success" = "Nous avons envoyé l'email de réinitialisation du mot de passe à \"%@\".";
+"sign_in_with_api_key" = "Se connecter avec la clé API";
+"enter_api_key" = "Entrez la clé API";
+"enter_api_key_explication" = "Pour gérer votre clé API, vous devez vous connecter à SimpleLogin via un navigateur puis cliquer sur le nom de votre profil.";
+"set_api_key" = "Définir la clé API";
+"change_api_url" = "Changer l'URL de l'API";
+"change_api_url_warning" = "NE PAS modifier l'URL de l'API, sauf si vous hébergez SimpleLogin sur votre propre serveur. Dans la plupart des cas, il est préférable de laisser ce champ par défaut.";
+"change_api_url_default_value" = "La valeur par défaut est https://app.simplelogin.io";
+"current_api_url" = "URL actuelle de l'API";
+"empty_api_url" = "L'URL de l'API est vide";
+"apply_change" = "Appliquer le changement";
+"reset_to_default_value" = "Remettre la valeur par défaut";
+"changed_api_url_to" = "Changement de l'URL de l'API en %@";
+"reset_api_url_to" = "Réinitialiser l'URL de l'API à https://app.simplelogin.io";
+ // Sign up 
+"sign_up" = "S'inscrire";
+"sign_up_for_simplelogin" = "S'inscrire à SimpleLogin";
+"sign_up_success" = "Vérifiez votre boîte de réception pour le code de vérification";
+"minimum_8_characters" = "Minimum 8 caractères";
 // Verification
-"enter_otp" = "Enter OTP";
-"enter_activation_code" = "Enter activation code";
-"paste_and_verify" = "Paste & verify";
-"code_found_from_the_clipboard" = "\"%@\" is found from the clipboard";
-"wrong_code_too_many_times" = "Wrong code too many times";
-"send_new_activation_code" = "We will send you a new activation code for \"%@\"";
-"check_new_verification_code" = "Check your inbox for new verification code";
+"enter_otp" = "Entrez le code OTP";
+"enter_activation_code" = "Entrez le code d'activation";
+"paste_and_verify" = "Coller et vérifier";
+"code_found_from_the_clipboard" = "\"%@\" a été trouvé dans le presse-papiers";
+"wrong_code_too_many_times" = "Mauvais code saisi trop souvent";
+"send_new_activation_code" = "Nous vous enverrons un nouveau code d'activation pour \"%@\".";
+"check_new_verification_code" = "Vérifiez votre boîte de réception pour le nouveau code de vérification";
 // Time
-<strind name="created_on" = "Created on";
-"just_now" = "just now";
-"minutes_ago" = "minutes ago";
-"day_ago" = "day ago";
-"days_ago" = "days ago";
-"week_ago" = "week ago";
-"weeks_ago" = "weeks ago";
-"month_ago" = "month ago";
-"months_ago" = "months ago";
-"year_ago" = "year ago";
-"years_ago" = "years ago";
+"created_on" = "Créé le";
+"just_now" = "à l'instant";
+"minutes_ago" = "il y a %@ minutes";
+"day_ago" = "il y a un jour";
+"days_ago" = "il y a %@ jours";
+"week_ago" = "il y a une semaine";
+"weeks_ago" = "il y a %@ semaines";
+"month_ago" = "il y a un mois";
+"months_ago" = "il y a %@ mois";
+"year_ago" = "il y a un an";
+"years_ago" = "il y a %@ ans";
```

Comment:
- Parameter must be added for plural strings
- Maybe replace `un` by `1` or by a parameter ?